src/context/mod.rs

```diff
@@ -286,3 +286,36 @@ impl Context {
         unsafe { raw::RedisModule_SetModuleOptions.unwrap()(self.ctx, options.bits()) };
     }
 }
+
+pub struct InfoContext {
+    pub ctx: *mut raw::RedisModuleInfoCtx,
```

Comment:
@oshadmi why `pub`?

Reply:
It is used outside, similar to `Context`, e.g., used by other modules, such as RedisJSON, etc.